### PR TITLE
Fix sync status update issue in Protected Application page

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2282834

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
image

File: packages/mco/components/protected-applications/utils.tsx [hunk 1]
```diff
@@ -88,8 +89,12 @@ export const isCleanupPending = (drpc: DRPlacementControlKind): boolean =>
 
 export const getReplicationHealth = (
   lastSyncTime: string,
-  schedulingInterval: string
+  schedulingInterval: string,
+  replicationType: REPLICATION_TYPE
 ): VOLUME_REPLICATION_HEALTH => {
+  if (replicationType === REPLICATION_TYPE.ASYNC) {
```
Comment (GowthamShanmugam) [inline packages/mco/components/protected-applications/utils.tsx:95]:
```suggestion
  if (replicationType === REPLICATION_TYPE.SYNC) {
```

File: packages/mco/components/mco-dashboard/disaster-recovery/cluster-app-card/cluster.tsx [hunk 2]
```diff
@@ -76,14 +77,16 @@ const checkVolumeReplicationHealth = (
 const checkKubeObjBackupHealth = (
   protectedAppMap: ProtectedAppsMap
 ): boolean => {
-  const kubeObjectLastTransitionTime =
-    protectedAppMap.placementControlInfo[0].kubeObjectLastTransitionTime;
+  const kubeObjectLastProtectionTime =
+    protectedAppMap.placementControlInfo[0].kubeObjectLastProtectionTime;
```
Comment (GowthamShanmugam) [inline packages/mco/components/mco-dashboard/disaster-recovery/cluster-app-card/cluster.tsx:81]:
```suggestion
  const {kubeObjectLastProtectionTime,  replicationType} =
    protectedAppMap.placementControlInfo[0];
```

File: packages/mco/types/dashboard.ts [hunk 3]
```diff
@@ -29,7 +29,7 @@ export type PlacementControlInfo = Partial<{
   subscriptions?: string[];
   // Only applicable for Discovered type
   // ClusterDataProtected condition transition time
```
Comment (GowthamShanmugam) [inline packages/mco/types/dashboard.ts:31]:
```suggestion
  // The time of the most recent successful kube object protection
```

Comment (GowthamShanmugam) [inline packages/mco/types/dashboard.ts:31]:
```suggestion
 
```

File: packages/mco/types/ramen.ts [hunk 4]
```diff
@@ -72,6 +72,7 @@ export type DRPlacementControlKind = K8sResourceCommon & {
     protectedNamespaces?: string[];
   };
   status?: {
+    lastKubeObjectProtectionTime?: string;
```
Comment (GowthamShanmugam) [inline packages/mco/types/ramen.ts:75]:
```suggestion
    // The time of the most recent successful kube object protection
    lastKubeObjectProtectionTime?: string;
```

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**